feat: hosted ui for flutter (#2864)
aws-amplify/docs: docs/lib/auth/fragments/flutter/signin_web_ui/20_platform_specific_setup.md

## How It Works

> Signin with web UI requires the Amplify plugin to display the signin UI inside a webview. This is managed internally by amplify-android and amplify-ios. After the signin process is complete they will redirect back to your app.

Reviewer: Not sure if we should add the following

>This is managed internally by amplify-android and amplify-ios. After the signin process is complete they will redirect back to your app

Author: ![image](https://user-images.githubusercontent.com/10621276/105774471-246f6280-5f1a-11eb-9881-6a796cb764df.png)

Author: I edited as we discussed.

> Signin with web UI requires the Amplify plugin to display the signin UI inside a webview. This is managed internally by amplify-android and amplify-ios. After the signin process is complete they will redirect back to your app.

Reviewer 2: ```suggestion
Sign-in with web UI requires the Amplify plugin to display the sign-in UI inside a web view. The UI is managed internally by the Amplify iOS and Android libraries. After the sign-in process is complete, the native libraries will redirect back to your app.
```

## Android Platform Setup

> Add the following activity to the `AndroidManifest.xml` file in your app's `Android` directory, replacing `myapp` with whatever value you used for your redirect URI prefix:

Reviewer 2: Should this be lowercase for a directory? `android`?

```suggestion
Add the following activity to the `AndroidManifest.xml` file in your app's `android` directory. Replacing `myapp` with the "redirect signin URI" you provided to the CLI:
```

Author: Pushed a similar revision from my local.

  ```xml
  <activity
      android:name="com.amazonaws.mobileconnectors.cognitoauth.activities.CustomTabsRedirectActivity">
      <intent-filter>
          <action android:name="android.intent.action.VIEW" />

          <category android:name="android.intent.category.DEFAULT" />
          <category android:name="android.intent.category.BROWSABLE" />

          <data android:scheme="myapp" />
      </intent-filter>
  </activity>
  ```

## iOS Platform Setup

> Add the following entry to the URL scheme in the `Info.plist` file in your app's `iOS` directory, replacing `myapp` with whatever value you used for your redirect URI prefix:

Reviewer 2: Same question -- is the directory really stylized with some upper case letters, or is it `ios`?
```suggestion
Add the following entry to the URL scheme in the `Info.plist` file in your app's `iOS` directory. Replace `myapp` with the "redirect signin URI" you provided to the CLI:
```

Author: Pushing similar revision from local
